components/config/examples/generate_sublime.rs

```diff
@@ -26,10 +29,31 @@ fn main() {
             builder.add_from_folder(package_dir, true).unwrap();
             let ss = builder.build();
             dump_to_file(&ss, packpath_newlines).unwrap();
-
-            for s in ss.syntaxes() {
-                if !s.file_extensions.is_empty() {
-                    println!("- {} -> {:?}", s.name, s.file_extensions);
+            let mut syntaxes: HashMap<String, SyntaxReference> = HashMap::new();
```

Comment:
I think Rust should be able to infer the type?

Reply:
Indeed. Thank you !